sdk/python/kfp/dsl/_container_op.py

```diff
@@ -644,7 +644,7 @@ class BaseOp(object):
     # in the compilation process to generate the DAGs and task io parameters.
     attrs_with_pipelineparams = [
         'node_selector', 'volumes', 'pod_annotations', 'pod_labels',
-        'num_retries', 'sidecars'
+        'num_retries', 'sidecars', 'tolerations'
```

Comment:
Are you sure? What's the use case when you want to use parameters in tolerations?

Reply:
I guess it is the same case with the node_selector, num_retries.
I think it is possible for users to use parameters for these during execution.